run_benchmark.sh

```diff
@@ -174,10 +174,10 @@ set_file_permissions() {
         fi
         # Attempt to handle execute permissions
         # as each directory/step should have a run.sh file
-        if [[ ! -x "$run_script" ]] && [[ ! $(chmod u+x "$run_script") ]]; then
-            echo "Unable to set execution for $run_script" >&2
-            exit 1
-        fi
+        #if [[ ! -x "$run_script" ]] && [[ ! $(chmod u+x "$run_script") ]]; then
+        #    echo "Unable to set execution for $run_script" >&2
+        #    exit 1
+        #fi
```

Comment:
This following PR changes this to an error message: https://github.com/EnterpriseDB/edb-benchmarks/pull/74

If you could uncomment as it is causing a merge conflict.
```suggestion
        if [[ ! -x "$run_script" ]] && [[ ! $(chmod u+x "$run_script") ]]; then
            echo "Unable to set execution for $run_script" >&2
            exit 1
        fi
```

Reply:
Un-commented